ui-tests/test/general.test.ts

```diff
@@ -0,0 +1,21 @@
+// Copyright (c) JupyterLite Contributors
+// Distributed under the terms of the Modified BSD License.
+
+import { test } from '@jupyterlab/galata';
+
+import { expect } from '@playwright/test';
+
+import { firefoxWaitForApplication } from './utils';
+
+test.use({ waitForApplication: firefoxWaitForApplication });
+
+test.describe('General', () => {
+  test.beforeEach(async ({ page }) => {
+    await page.goto('lab/index.html');
+  });
+
+  test('Kernel is registered', async ({ page }) => {
+    const imageName = 'launcher.png';
+    expect(await page.screenshot()).toMatchSnapshot(imageName);
```

Comment:
Should we reduce the scope of the screenshot and just take the XPython item in the UI? That way the test is less likely to fail in the futur.

Reply:
Maybe just the launcher then?

Reply:
We could even just make sure the test XPython is in the UI and. Not take a screenshot.

Reply:
Right. Although with a screenshot we can also check it appears under the Console section, and its display name is correct (but these could also be regular expects).